src/components/expression/expression-renderer.js

```diff
@@ -551,6 +713,9 @@ genotet.ExpressionRenderer.prototype.drawMatrixCells_ = function() {
   var heatmapRects = heatmapRows.selectAll('rect').data(_.identity);
   heatmapRects.enter().append('rect');
   heatmapRects
+    .attr('class', function(value, i, j) {
+      return 'cell row' + j + ' column' + i;
```

Comment:
In heatmap content, there are not only cell rect, but also the drag selection rect. If I want to select all the cell rect, we can add cell class to every rect.